desktop-app/src/renderer/components/Previewer/Device/index.tsx

```diff
@@ -268,11 +284,13 @@ const Device = ({ isPrimary, device }: Props) => {
 
     if (!isPrimary) {
       setTimeout(() => {
-        window.electron.ipcRenderer.invoke<
-          DisableDefaultWindowOpenHandlerArgs,
-          DisableDefaultWindowOpenHandlerResult
-        >('disable-default-window-open-handler', {
-          webContentsId: webview.getWebContentsId(),
+        webview.addEventListener('dom-ready', () => {
```

Comment:
Any specific reason behind this change?

Reply:
There is an issue caused when users repeatedly clicked the individual layout button, thereby exiting from the individual layout mode and entering the column or flex layout mode. When this happens, the attached setTimeout is called repeatedly and 'webContentsId' returns an error value.
I tried fixing this by clearing the setTimeout and flushing the value out but yet this kept happening.
Hence, have attached the listener 'dom-ready'.

Attaching the error message below:
<img width="488" alt="Screenshot 2023-05-28 at 6 06 06 PM" src="https://github.com/responsively-org/responsively-app/assets/59393936/86c52eec-46ae-4651-8e75-d197f3fe4fd5">

Reply:
OK cool, as long as we don't open multi windows when a `target="_blank"` link is clicked this is fine.